sdk/ai/azure-ai-projects/CHANGELOG.md

```diff
@@ -4,10 +4,18 @@
 
 ### Features added
```

Comment:
I would also set release date to today (?) or Tuesday.

Reply:
done